scripts/check_doc_ipfs_hashes.py

```diff
@@ -30,7 +30,7 @@
 
 
 IPFS_HASH_REGEX = r"bafybei[A-Za-z0-9]{52}"
-AEA_COMMAND_REGEX = r"(?P<full_cmd>(?P<cli>aea|autonomy) (?P<cmd>fetch|add .*) (?:(?P<vendor>.*)\/(?P<package>.[^:]*):(?P<version>\d+\.\d+\.\d+)?:?)?(?P<hash>bafybei[A-Za-z0-9]{52}))"
+AEA_COMMAND_REGEX = fr"(?P<full_cmd>(?P<cli>aea|autonomy) (?P<cmd>.*) (?:(?P<vendor>.*)\/(?P<package>.[^:]*):(?P<version>\d+\.\d+\.\d+)?:?)?(?P<hash>{IPFS_HASH_REGEX}))"
```

Comment:
Using a f-regex-string now to avoid regex duplication

Reply:
You can use the public id regex here

Reply:
Ok, I've incorporated some already existing regexes but since i needed capturing groups i still had to add some custom ones.